### DISCUSSION TITLE
Installing DirectoryLister to another directory

### DISCUSSION THREAD

**Discussion by iYonga** [DirectoryLister/DirectoryLister#612] [Help & Support]
![image](https://user-images.githubusercontent.com/30579752/114313879-0967c400-9b01-11eb-96bc-fc82b0600adf.png)

It worked well in the C drive but I can't move it to D now. What to do?

**Comment by PHLAK** [comment]
See https://docs.directorylister.com/help-and-support/common-issues#open_basedir-restriction-in-effect.

You must install Directory Lister in the folder you wish to list.
